CHANGELOG.md

```diff
@@ -1,6 +1,21 @@
 # Change Log
 All notable changes to the "azure-appservice" extension will be documented in this file.
 
+## 0.9.0 - 2018-07-12
+### Added
+- Remote file editting for Web Apps and slots (previously in preview)
```

Comment:
How about '(previously required a feature flag)'

'Preview' is confusing because the extension is still in preview